homeassistant/components/zha/websocket_api.py

```diff
@@ -1204,6 +1209,25 @@ async def websocket_restore_network_backup(
         connection.send_result(msg[ID])
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zha/network/change_channel",
+        vol.Required(ATTR_NEW_CHANNEL): vol.Any(
+            "auto", vol.All(cv.positive_int, vol.Range(11, 26))
```

Comment:
You don't need positive int, as that's checked by vol.Range
```suggestion
            "auto", vol.Range(11, 26)
```